API/AWMPS_API.json

```
@@ -0,0 +1,79 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "Your API",
+    "version": "1.0.0"
+  },
+  "paths": {
+    /pot-types {
+      "get": {
+        "summary": "Get a list of all pot types",
+        "responses": {
+          "200": {
+            "description": "Successful response"
+          }
+        }
+      },
+      "post": {
+        "summary": "Create a new pot type",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "type": "string"
+                  },
+                  "capacity": {
+                    "type": "integer"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "responses": {
+          "201": {
+            "description": "Pot Type created successfully"
+          }
+        }
+      }
+    },
+    "/api/pot-types/{id}": {
```

missing `id` parameter definition:
```suggestion
"/api/pot-types/{id}": {
"parameters": [
        {
          "name": "id",
          "in": "path",
          "required": true,
          "schema": {
            "type": "string"
          },
          "description": "ID of the pot type"
        }
      ],
```